cypress/integration/login.js

```diff
@@ -21,4 +21,8 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
     })
+
+    it('Test current date', () => {
+        cy.get('[data-test="timesheets-current-day"]').find('span').contains(13)
```

Comment:
Does this selector only finds one element, or it will find a whole bunch of elements? Also, will your test work tomorrow?